package.json

```diff
@@ -2,50 +2,65 @@
   "name": "block-lab",
   "title": "Block Lab",
   "version": "1.5.2",
+  "description": "WordPress plugin with a simple templating system for building custom blocks.",
```

Comment:
Feel free to suggest something different, I added this because there was an error from not having a description.